src/runtime/contrib/dnnl/dnnl_json_runtime.cc

```diff
@@ -298,15 +743,35 @@ class DNNLJSONRuntime : public JSONRuntimeBase {
     // has not yet been bound to the other DNNL memory; otherwise it may have memory leak.
     ICHECK_EQ(entry_out_mem_.count(eid), 0);
 
-    // TODO(@comanic): Support other data types (i.e., int8).
-    auto data_node = nodes_[entry.id_];
-    auto dltype = data_node.GetOpDataType()[entry.index_];
-    ICHECK_EQ(dltype.bits, 32);
-
     entry_out_mem_[eid] = {mem, offset};
     return entry_out_mem_[eid].first;
   }
 
+  dt dtype_dl2dnnl(DLDataType dltype) {
```

Comment:
duplicated func as `query_layout.cc`?

Reply:
Yes, I know that but couldn't find a better way.
Here are the path of the two source files:
```
src/runtime/contrib/dnnl/dnnl_json_runtime.cc
src/relay/backend/contrib/dnnl/query_layout.cc
```
I'm thinking about add a header like:
`src/runtime/contrib/dnnl/dnnl_utils.h`
and put the functions there, then add 
`#include "../../../../runtime/contrib/dnnl/dnnl_utils.h`
in `query_layout.cc`, which is a little dirty.
Do you have better idea, or should we just ignore this duplication?

Reply:
I think we'd better to avoid creating duplicated codes in a project (which might not be consistent as the developing goes on), it's ok to move it into a header file and include it via relative path, see the following:
```
./src/relay/backend/contrib/ethosu/source_module.cc:43:#include "../../../../runtime/file_utils.h"
./src/relay/backend/contrib/dnnl/codegen.cc:39:#include "../../../../runtime/contrib/json/json_node.h"
./src/relay/backend/contrib/verilator/codegen.cc:36:#include "../../../../runtime/contrib/json/json_node.h"
./src/relay/backend/contrib/verilator/codegen.cc:37:#include "../../../../runtime/contrib/verilator/verilator_runtime.h"
./src/relay/backend/contrib/cmsisnn/tir_to_runtime.cc:26:#include "../../../../runtime/file_utils.h"
./src/relay/backend/contrib/bnns/codegen.cc:33:#include "../../../../runtime/contrib/json/json_node.h"
./src/relay/backend/contrib/ethosn/codegen_ethosn.h:46:#include "../../../../runtime/contrib/ethosn/ethosn_runtime.h"
./src/relay/backend/contrib/codegen_json/codegen_json.h:38:#include "../../../../runtime/contrib/json/json_node.h"
./src/relay/backend/contrib/codegen_json/codegen_json.h:39:#include "../../../../runtime/contrib/json/json_runtime.h"
```

Reply:
Agree, it's better for long term development. I will add a `dnnl_utils.h` to hold the duplicated codes, include but not limited to the function `dtype_dl2dnnl()`.

Reply:
Done.